site/app/templates/generic/Popup.twig

```diff
@@ -15,17 +15,17 @@
                 {% block body_panel %}
                     <div class="form-body">
                         {% block body %}Be sure to override the body block.{% endblock %}
-                    </div>
-                {% endblock %}
-                {% block buttons_panel %}
-                    <div class="form-buttons">
-                        <div class="form-button-container">
-                            {% block buttons %}
-                                {% block close_button %}
-                                   <a href="javascript:closePopup('{{ block('popup_id') }}')" class="btn btn-default close-button">Close</a>
-                                {% endblock %}
-                            {% endblock %}
-                        </div>
+                        {% block buttons_panel %}
+                          <div class="form-buttons">
+                                <div class="form-button-container">
```

Comment:
This works well. The only comment I would make is that the close and submit buttons overlap slightly with the bottom of the modal (see image below). You can fix that with some css on `#form-button-container`. `padding-bottom:.5rem` would fix this issue and it looks better.
![image](https://user-images.githubusercontent.com/18558130/74043815-125a8400-4998-11ea-9330-77ce4dd0a17a.png)

Reply:
Added some changes please take another look, Thanks!